flink-connector-kafka/src/main/java/org/apache/flink/connector/kafka/source/enumerator/KafkaSourceEnumState.java

```diff
@@ -42,7 +42,7 @@ public KafkaSourceEnumState(
         this.initialDiscoveryFinished = initialDiscoveryFinished;
     }
 
-    KafkaSourceEnumState(
+    public KafkaSourceEnumState(
```

Comment:
This is the only KafkaSource code that I had to modify. As a result of moving packages, this needed to be publicly visible (the class is still annotated as Internal)